### PR TITLE
update dialog padding and close icon

### PR DESCRIPTION
**Pull request by storywithoutend**

### CODE REVIEW

File: components/src/components/organisms/Dialog/Dialog.tsx [hunk 1]
```diff
@@ -36,19 +43,21 @@ const StyledCard = styled.div(
   ({ theme }) => css`
     display: flex;
     flex-direction: column;
-    align-items: stretch;
-    gap: ${theme.space['6']};
-
-    padding: ${theme.space['6']};
+    align-items: center;
+    gap: ${theme.space['4']};
+    padding: ${theme.space['4']};
```
Comment (TateB) [inline components/src/components/organisms/Dialog/Dialog.tsx:48]: dont we still use 24px padding on mobile dialogs?

**Review by TateB — CHANGES_REQUESTED** [review]

File: components/src/components/organisms/Dialog/Dialog.tsx (hunk 1 above)
Comment (storywithoutend) [reply components/src/components/organisms/Dialog/Dialog.tsx:48]: https://www.figma.com/file/iP5p7Vd2C1VhRjTXizfERU/Thorin?node-id=690%3A9627&t=AEB5HNkT7r4o8IIq-0
16px on mobile
24px on desktop

Comment (TateB) [reply components/src/components/organisms/Dialog/Dialog.tsx:48]: i guess the transaction modal has different padding then... weird

**Review by TateB — APPROVED** [review]

File: components/src/components/organisms/Dialog/Dialog.tsx (hunk 1 above)
Comment (storywithoutend) [reply components/src/components/organisms/Dialog/Dialog.tsx:48]: I think it's been inconsistently applied. I asked for standardized format yesterday at the meeting.

**Pull request merged**